RadarSDK/Radar.m

```diff
@@ -111,6 +114,58 @@ + (void)startTrackingWithOptions:(RadarTrackingOptions *)options {
     [[RadarLocationManager sharedInstance] startTrackingWithOptions:options];
 }
 
++ (void)mockTrackingWithOrigin:(CLLocation *)origin
+                   destination:(CLLocation *)destination
+                          mode:(RadarRouteMode)mode
+                        points:(int)points
+                      interval:(NSTimeInterval)interval
+             completionHandler:(RadarTrackCompletionHandler _Nullable)completionHandler {
+    [[RadarAPIClient sharedInstance] getMockFromOrigin:origin
+                                           destination:destination
+                                                  mode:mode
+                                                points:points
+                                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res, NSArray<RadarCoordinate *> *_Nullable coordinates) {
+                                         if (status != RadarStatusSuccess || !coordinates) {
+                                             if (completionHandler) {
+                                                 completionHandler(status, nil, nil, nil);
+                                             }
+
+                                             return;
+                                         }
+
+                                         NSTimeInterval intervalLimit = interval;
+                                         if (intervalLimit < 2) {
+                                             intervalLimit = 2;
+                                         } else if (intervalLimit > 60) {
+                                             intervalLimit = 60;
+                                         }
+
+                                         for (int i = 0; i < coordinates.count; i++) {
+                                             RadarCoordinate *coordinate = coordinates[i];
+                                             __block CLLocation *location = [[CLLocation alloc] initWithCoordinate:coordinate.coordinate
+                                                                                                          altitude:-1
+                                                                                                horizontalAccuracy:5
+                                                                                                  verticalAccuracy:-1
+                                                                                                         timestamp:[NSDate new]];
```

Comment:
nit: do we need to give caller more flexibility to specify these additional params (we could use these values as a preset of course)?

Reply:
You mean `altitude`, `horizontalAccuracy`, and `verticalAccuracy`? I could see the server returning `altitude`, but most customers probably won't need it. The iOS simulator always uses `horizontalAccuracy = 5` for locations in the simulator, just following the same convention here.